models/src/main/protobuf/RhoTypes.proto

```diff
@@ -149,7 +153,8 @@ message New {
     Par p = 2 [(scalapb.field).no_box = true];
     // For normalization, uri-referenced variables come at the end, and in lexicographical order.
     repeated string uri = 3;
-    bytes locallyFree = 4 [(scalapb.field).type = "coop.rchain.models.AlwaysEqual[scala.collection.immutable.BitSet]"];
+    DeployerAuth auth = 4;
```

Comment:
We should make sure this is only filled when there's a `rho:deployer:auth` in the `uri`-s. Otherwise this field should stay empty (all fields are optional in proto 3)

Reply:
I mention the above is just in case :point_up:

Reply:
also, let's call it `deployerId`

Reply:
Done